tests/baselines/reference/declarationEmit_UnknownImport.errors.txt

```diff
@@ -1,14 +1,17 @@
 tests/cases/compiler/declarationEmit_UnknownImport.ts(2,1): error TS2304: Cannot find name 'SomeNonExistingName'.
+tests/cases/compiler/declarationEmit_UnknownImport.ts(2,14): error TS2304: Cannot find name 'SomeNonExistingName'.
 tests/cases/compiler/declarationEmit_UnknownImport.ts(2,14): error TS2503: Cannot find namespace 'SomeNonExistingName'.
 tests/cases/compiler/declarationEmit_UnknownImport.ts(2,14): error TS4000: Import declaration 'Foo' is using private name 'SomeNonExistingName'.
 
 
-==== tests/cases/compiler/declarationEmit_UnknownImport.ts (3 errors) ====
+==== tests/cases/compiler/declarationEmit_UnknownImport.ts (4 errors) ====
     
     import Foo = SomeNonExistingName
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 !!! error TS2304: Cannot find name 'SomeNonExistingName'.
                  ~~~~~~~~~~~~~~~~~~~
+!!! error TS2304: Cannot find name 'SomeNonExistingName'.

```

Comment:
@vladima, i removed the older `Cannot find name` error, as it was generated during declaration emit, and was not really needed. the new one however, is produced as we now call checkExpression to make sure we mark the identifier. i do not think there is an easy way to do that, unless we wire in a flag into checkExpression functions to not report errors. i do not think it is worth the complexity though.